README.md

```diff
@@ -35,21 +35,41 @@ And more to come, see Marker's goals and limitations below.
 
 The following is an abbreviated guide. Check out [The Marker Book] for detailed instructions and additional information.
 
-[The Marker Book]: rust-marker.github.io/marker/book
+[The Marker Book]: https://rust-marker.github.io/marker/book
 
 ### Installation
 
+#### Download pre-compiled binaries (recommended)
+
+We provide pre-compiled binaries for the mainstream platforms. See the list of available artifacts in our [Github Releases](https://github.com/rust-marker/marker/releases/latest).
+
+Run the following bash script to install the required rust toolchain dependency on your machine and download the current version of `cargo-marker` CLI, and the internal driver.
```

Comment:
I also like having this in the readme, I think it's fine to keep them in sync. I bet there is a tool that allows synchronizing regions of text somewhere out there similar to our custom `replace-version`, but I hope this problem will be a rare occasion.

Actually, I would really like to have windows support for the install script. I just didn't make that effort due to laziness, powershell is even worse than bash because I don't ever write it. I wish we could use a cross-platform shell like nutshell, but that one isn't at `1.0` yet, and isn't installed by default on windows and UNIX, unfortunately.

I'll try to add a PowerShell install script impl, but in a separate PR, I'd like to merge this one without that to not make "the ideal" the enemy of "the good". But for now, I'll add that clarification